lib/instance.js

```diff
@@ -482,6 +482,8 @@ Instance.prototype._setInclude = function(key, value, options) {
  *
  * @param {Object} [options]
  * @param {string[]} [options.fields] An optional array of strings, representing database columns. If fields is provided, only those columns will be validated and saved.
+ * @param {string[]} [options.where] An optional object with where conditions.
```

Comment:
Hmm, not sure I like this - `instance.save` is meant to save a single row, which should be identified by primary key - I think adding a where argument here as well might confuse new users as to what the method does. I think this feature fits better in `model.update`